src/electionguard/serializable.py

```diff
@@ -30,10 +36,11 @@ def to_json(self, strip_privates: bool = True) -> str:
         :param strip_privates: strip private variables
         :return: the json representation of this object
         """
+        set_serializers()
```

Comment:
These were moved into here to reduce turning these on every time they needed to be used.